src/systems/ec/implementations/react/components/table/stories/Index.jsx

```diff
@@ -1,14 +1,38 @@
+/* eslint-disable react/react-in-jsx-scope, import/no-extraneous-dependencies */
+import React from 'react';
+import StoryWrapper from '@ecl/story-wrapper';
 import Table from '../examples/Table';
 import TableZebra from '../examples/TableZebra';
 import TableMulti from '../examples/TableMulti';
+import TableSort from '../examples/TableSort';
 
 export default {
   title: 'Components/Table',
+
+  decorators: [
+    story => (
+      <StoryWrapper
+        afterMount={() => {
+          if (!window.ECL) return {};
+
+          const autoinit = window.ECL.autoInit();
+          return { components: autoinit.components };
+        }}
+        beforeUnmount={context => {
+          if (context.components) {
+            context.components.forEach(c => c.destroy());
+          }
+        }}
+      >
+        {story()}
+      </StoryWrapper>
+    ),
+  ],
 };
 
-export const Default = Table;
+export const DefaultTable = Table;
 
-Default.story = {
+DefaultTable.story = {
```

Comment:
The `Default` here is on purpose, it is consistently such throughout all stories in order to keep stories' paths compatible with ecl-twig's comparison tool which scrapes such paths https://5ed9193f8dee528866f37b7e--europa-component-library.netlify.app/playground/ec/?path=/story/components-table--default which now would become https://5ed9193f8dee528866f37b7e--europa-component-library.netlify.app/playground/ec/?path=/story/components-table--default-table and break the patter.

Please revert this change.